example/src/main/scala/example/libs/zio/kafka/SocketConsumer.scala

```diff
@@ -0,0 +1,44 @@
+package example.libs.zio.kafka
+
+import zhttp.http._
+import zhttp.service._
+import zhttp.socket._
+import zio._
+import zio.kafka.consumer.{Consumer, ConsumerSettings, Subscription}
+import zio.kafka.serde.Serde
+import zio.stream.ZStream
+import zio.system._
+
+object SocketConsumer extends zio.App {
+  override def run(args: List[String]): URIO[zio.ZEnv, ExitCode] = {
+
+    envs.flatMap { env =>
+      val settings = ConsumerSettings(List(env("KAFKA_BOOTSTRAP_SERVER")))
+        .withGroupId(env("KAFKA_GROUP_ID"))
+
+      val consumer = ZLayer.fromManaged(Consumer.make(settings))
+
+      Server
+        .start(env("ZHTTP_PORT").toInt, AllApis())
+        .provideCustomLayer(consumer)
+    }
+  }.exitCode
+}
+
+object AllApis {
```

Comment:
Follow consistent pattern as followed in other examples. You need not to create object AllApi's